plugin/trino-ranger/pom.xml

```diff
@@ -0,0 +1,257 @@
+        <dep.hbase.jetty.version>9.4.44.v20210927</dep.hbase.jetty.version>
+        <dep.commons.cli.version>1.4</dep.commons.cli.version>
+        <dep.apache-httpclient.version>4.5.10</dep.apache-httpclient.version>
+        <dep.jersey.version>1.19.3</dep.jersey.version>
+        <dep.codehaus.jackson.version>1.9.13</dep.codehaus.jackson.version>
+        <dep.javax.servlet.version>2.5</dep.javax.servlet.version>
+        <dep.kafka.version>2.5.0</dep.kafka.version>
+        <dep.carrotsearch-hppc.version>0.8.1</dep.carrotsearch-hppc.version>
+        <dep.apache-httpasyncclient.version>4.1.4</dep.apache-httpasyncclient.version>
+        <dep.apache-httpcore.version>4.4.12</dep.apache-httpcore.version>
+        <deps.junit.version>4.13.2</deps.junit.version>
+        <deps.commons-lang.version>2.6</deps.commons-lang.version>
+        <deps.google-code-gson.version>2.2.4</deps.google-code-gson.version>
+        <deps.jna.version>5.8.0</deps.jna.version>
+    </properties>
+
+    <dependencies>
+
+        <dependency>
+            <groupId>io.trino</groupId>
+            <artifactId>trino-hadoop-toolkit</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>io.trino.hadoop</groupId>
+            <artifactId>hadoop-apache</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>io.airlift</groupId>
+            <artifactId>bootstrap</artifactId>
+            <version>${dep.airlift.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>io.airlift</groupId>
+            <artifactId>configuration</artifactId>
+            <version>${dep.airlift.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>io.airlift</groupId>
+            <artifactId>log</artifactId>
+            <version>${dep.airlift.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>com.google.inject</groupId>
+            <artifactId>guice</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>commons-lang</groupId>
+            <artifactId>commons-lang</artifactId>
+            <version>${deps.commons-lang.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>javax.inject</groupId>
+            <artifactId>javax.inject</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.ranger</groupId>
+            <artifactId>ranger-plugin-classloader</artifactId>
+            <version>${dep.ranger.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.ranger</groupId>
+            <artifactId>ranger-plugins-common</artifactId>
+            <version>${dep.ranger.version}</version>
+            <exclusions>
+                <exclusion>
```

Comment:
Are you sure that things are still working after exclusions? It looks risky. I think we we should shade libraries instead otherwise we may get runtime errors that class is not present.

Reply:
@dprophet ^^ This is also something that we will validate with product tests.

Reply:
Yes. This works after exclusions. If you do a mvn dependency:tree you will see a deep tree where same dependencies (with different) versions exists.

bitsondatadev is correct. This can be ironed out with automated product tests.